lib/ansible/modules/network/nxos/nxos_vlan.py

```diff
@@ -458,7 +458,7 @@ def parse_interfaces(module, vlan):
     vlan_int = []
     interfaces = vlan.get('vlanshowplist-ifidx')
     if interfaces:
-        for i in interfaces.split(','):
+        for i in interfaces:
```

Comment:
`interfaces` cannot be a list. The key `'vlanshowplist-ifidx'` returns string only.